lib/Doctrine/DBAL/Driver/PDOPgSql/Driver.php

```diff
@@ -88,11 +88,8 @@ private function _constructPdoDsn(array $params)
 
         if (isset($params['dbname'])) {
             $dsn .= 'dbname=' . $params['dbname'] . ' ';
-        } else {
-            // Used for temporary connections to allow operations like dropping the database currently connected to.
-            // Connecting without an explicit database does not work, therefore "postgres" database is used
-            // as it is certainly present in every server setup. 
-            $dsn .= 'dbname=postgres' . ' ';
+        } elseif (isset($params['default_dbname'])) {
+            $dsn .= 'dbname=' . $params['default_dbname'] . ' ';
```

Comment:
We should still connect to `postgres` if the parameter is not set as we will have errors again otherwise.